api/server/server.go

```diff
@@ -1206,7 +1220,10 @@ func postBuild(eng *engine.Engine, version version.Version, w http.ResponseWrite
 		authConfig        = &registry.AuthConfig{}
 		configFileEncoded = r.Header.Get("X-Registry-Config")
 		configFile        = &registry.ConfigFile{}
-		job               = eng.Job("build")
+		buildConfig       = builder.NewBuildConfig()
+		stdout            = engine.NewOutput()
```

Comment:
I even realized that stderr wasn't really being used at all, so removed! Let me know what you think.